docker/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN pip install --upgrade \
 	"datadog==0.35.0" \
 	"docker-compose==1.29.1" \
 	"pyjwt>=1.6.0" \
+	"slack-sdk==3.21.3" \
```

Comment:
Do we really need a new dependency? Couldn't we just send an HTTP request to trigger a webhook? I remember you could've just sent an request for a simple message. But the SDK is probably needed because of the file upload right?

Reply:
Yes. Webhook cannot upload the files but SKD does.